Package.swift

```diff
@@ -33,7 +42,7 @@ let package = Package(
             exclude: ["Info.plist"]),
         .testTarget(
             name: "MapboxDirectionsTests",
-            dependencies: ["MapboxDirections"],
+            dependencies: ["MapboxDirections"] + optionalTestTargetDependecies,
```

Comment:
If you specify a [target](https://developer.apple.com/documentation/swift_packages/target/dependency/3583309-target) explicitly, then its condition can be [dependent on the platform](https://developer.apple.com/documentation/swift_packages/targetdependencycondition/3554520-when) without resorting to conditional compilation.

Reply:
thx, fixed this one